app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show 
+    @recipe = Recipe.find(params[:id])
+    @total_cost = @recipe.total_cost
```

Comment:
It's totally okay to have this @total_cost instance variable here, but also want to confirm that it's okay to call `@recipe.total_cost` in the view as well. You're not violating MVC by calling a model method. You encapsulating the AR logic involved in the calculation to a model method, but you can call that method from anywhere. Great work!